### ISSUE TITLE
No notification with Apple watch

### ISSUE THREAD

**Issue by oDEADPOOLo** [pwn20wndstuff/Undecimus#2128]
**Describe the bug**
After jail breaking on 5.3.1 uncover, Apple Watch no longer receives any notifications from apps or messages. Phone calls do work. But it no longer mirrors or pushes alerts to the watch. 

If phone is not jail broken, or a reboot happens it restores normal functions of the watch. I e tried airplane mode on only the phone, only the watch, both at the same time, unpairing/ re-pairing (before JB / and after JB). 

Had another test it and she had functionality until just yesterday. Prior to, she had normal functions till then. Now it’s stopped, almost as if a service had stopped working. Unsure. Will continue to test. 

**To Reproduce**

**Screenshots**

**Device (please complete the following information):**
 - iOS Version: 13.5
 - iOS Device: XS Max
 - unc0ver Version: 5.3.1

**Place an "x" between the brackets if true:**
 - [?] this is a bug others will be able to reproduce
 - [x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [?] this issue is present after a rootfs restore
 - [x] this issue is present on the latest version of unc0ver

**Logs**

**Comment by Hamlock-maneuverr** [comment]
put the watch into airplane mode and turn off bluetooth before you jailbreak.  turn back on after jailbroken.

**Comment by oDEADPOOLo** [comment]
@Hamlock-maneuverr, I’ve tried this method. Also have put the phone in airplane mode only. Have also put both the watch and phone in airplane mode as well. I’ve turned off Bluetooth from the settings app and not the CC. 
Also have tried turning off the the watch then back on once jailbroken. And have also unpaired then re-paired once jailbroken. 

Edit: Watch OS 6.2.8
Apple Watch Series 2

**Comment by Hamlock-maneuverr** [comment]
Did you turn off the "watches" bluetooth?  from inside the watch itself?

**Comment by oDEADPOOLo** [comment]
@Hamlock-maneuverr yes I have tried this. I recently tried to disable all tweaks that were active as well. Went into iCleaner and disabled them. And tried to toggle them all back on one by one. No luck though.

**Comment by oDEADPOOLo** [comment]
So if anyone has this same issue I think I’ve found a work around. 
#1: if you are not receive notifications properly to your Apple Watch. (ie. text messages or any notification will give your phone an audible alert and banner or notification, but not your watch). 
#2: you will need iCleaner pro and uncover jailbreak app for this to work. 
#3: when you notice the notifications are no longer working, you must reboot your phone to be able to re-jailbreak. (I test using google voice, sending myself a message and then immediately locking my phone to see if it formed to my watch).
#4: once rebooted, you must disable load tweaks in the uncover app. Then re-jailbreak. 
#5: once jailbroken, open iCleaner pro, go to advanced, then to Cydia substrate add one and click all and apply. This should make sure all tweaks or add one are disabled. Then you must run iCleaner pro analyze and clean. Or just clean. Respring afterwards. 
#6: after this I send myself a test message. Sometimes this is all you need. If it doesn’t work, reboot the phone again. Send a test message to ensure you have notifications. Re-jailbreak as before leaving tweaks disabled. Once jailbroken you run iCleaner pro analyze and clean again. Then respring. I send a test message here to myself. If all is well, and should be working....
#7: if all of this worked. And you now have working notifications going to your watch, I open up iCleaner pro again, this time back to advanced, then Cydia substrate add one, then click all, then apply. I once again run analyze and clean. Or just clean each time up to you. Once I respring the phone after iCleaner is done, I send yet another test message to confirm it’s working. 

With this method. I have successfully made sure my watch continues to work properly with my phone.
